### ISSUE TITLE
Cannot connect to Barrier from Win32 Synergy.

### ISSUE THREAD

**Issue by jflamy** [yupi2/synergy#13]
### Operating Systems ###

Server: Windows 11 Home
Client: Window 10 Home (32 bits)

### Synergy Version ###

Your fork as client, latest version.
Server is Barrier 2.3.3-3395

### Steps to reproduce bug ###

Try to connect to Barrier from the Win32 machine.
Fails to connect due to SSL Error.
Does not work even if server is started as barriers.exe --disable-crypo
Disabling SSL on the individual connection also does not work.

### Other info ###

No problems using Barrier with Raspberry Pi and other Windows 10 Barrier.

**Comment by yupi2** [comment]
If it's SSL/TLS related then it could be due to a cipher-suite mismatch since this fork uses libressl 2.7.4 which is over 3 years old. 
If it's not that, then there's potentially some protocol changes between this and barrier. I don't follow barrier to know if they've changed anything and I don't remember if I changed anything in this project.

Unfortunately, I don't have any desire to touch this project because synergy is a bad code-base and it's been years since I've used this.

Your best bet would be to hope barrier gets 32-bit support.
